src/main/java/moze_intel/projecte/network/PacketHandler.java

```diff
@@ -71,15 +71,15 @@ public static void sendFragmentedEmcPacketToAll()
 		}
 	}
 
-	private static int[][] serializeEmcData()
+	private static long[][] serializeEmcData()
 	{
-		int[][] ret = new int[EMCMapper.emc.size()][];
+		long[][] ret = new long[EMCMapper.emc.size()][];
 		int i = 0;
-		for (Map.Entry<SimpleStack, Integer> entry : EMCMapper.emc.entrySet())
+		for (Map.Entry<SimpleStack, Long> entry : EMCMapper.emc.entrySet())
 		{
 			SimpleStack stack = entry.getKey();
 			int id = Item.REGISTRY.getIDForObject(Item.REGISTRY.getObject(stack.id));
-			ret[i] = new int[] { id, stack.damage, entry.getValue() };
+			ret[i] = new long[] { id, stack.damage, entry.getValue() };
```

Comment:
I'm in favor of making this an actual object type so we don't have to waste space on the IDs, but I'll let it slide and fix it after merging this